### PR TITLE
Bump to citus_enterprise 10.2.6

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: .github/workflows/update_package_properties.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
         run: sudo apt-get update && sudo apt install libcurl4-openssl-dev libssl-dev python3-testresources
 
       - name: Clone tools branch
-        run: git clone -b v0.8.12 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b v0.8.14 --depth=1  https://github.com/citusdata/tools.git tools
```
Comment (hanefi) [inline .github/workflows/update_package_properties.yml]:
```suggestion
        run: git clone -b v0.8.14 --depth=1 https://github.com/citusdata/tools.git tools
```

File: packaging_ignore.yml [hunk 2]
```diff
@@ -15,6 +15,7 @@ base:
   - ".* unused substitution variable \\$\\{misc:Depends\\}.*"
   - ".*: substitution variable \\$\\{postgresql:Depends\\} unused, but is defined.*"
   - ".* unused substitution variable \\$\\{postgresql:Depends\\}.*"
+  - ".*: WARNING: unrecognized options: --with-security-flags"
```
Comment (hanefi) [inline packaging_ignore.yml]: I do not think we should ever ignore this line.

**Review by hanefi — CHANGES_REQUESTED** [review]

File: packaging_ignore.yml [hunk 3]
```diff
@@ -27,3 +27,4 @@ rpm:
   - ".*: W: unstripped-binary-or-object /usr/pgsql-.*/lib/citus.so"
   - ".*: W: unstripped-binary-or-object /usr/pgsql-.*/bin/pg_send_cancellation"
   - ".*: E: zero-length /usr/pgsql-.*/share/extension/citus.*"
+
```
Comment (hanefi) [inline packaging_ignore.yml]:
```suggestion
```

File: packaging_ignore.yml (hunk 2 above)
Comment (gurkanindibay) [reply packaging_ignore.yml]: You're right removed it

File: packaging_ignore.yml (hunk 3 above)
Comment (gurkanindibay) [reply packaging_ignore.yml]: Removed

**Comment by hanefi** [comment]
Do we want to disable nightly tests in a separate PR?

**Review by hanefi — APPROVED** [review]

**Pull request merged**